src/DIRAC/Resources/Catalog/RucioFileCatalogClient.py

```diff
@@ -704,10 +703,11 @@ def getFileUserMetadata(self, path):
         all its parents
         """
         resDict = {"Successful": {}, "Failed": {}}
+        path = next(iter(path))
         try:
             did = self.__getDidsFromLfn(path)
-            meta = [met for met in self.client.get_metadata_bulk(dids=[did], inherit=True)]
-            if meta["did_type"] == "FILE":  # Should we also return the metadata for the directories ?
+            meta = next(self.client.get_metadata_bulk(dids=[did], inherit=True, plugin="ALL"))
```

Comment:
Note: This (the `plugin="ALL"`) relies on a not-yet-merged PR in rucio:
https://github.com/rucio/rucio/pull/7317